### DISCUSSION TITLE
a potential error in the docs

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#4377] [Q&A]
https://github.com/FreshRSS/FreshRSS/tree/edge/Docker#quick-run
```
docker run -d --restart unless-stopped --log-opt max-size=10m \
  -p 8080:80 \
  -e TZ=Europe/Paris \
  -e 'CRON_MIN=1,31' \
  -v freshrss_data:/var/www/FreshRSS/data \
  --name freshrss \
  freshrss/freshrss
```

running outputs:

    docker: Error response from daemon: unknown log opt 'max-size' for journald log driver.

Removing  `--log-opt max-size=10m` resolves the issue.

**Comment by Frenzie** [comment]
Not an error exactly. Presumably your or my distro changed this from json-file to journald or vice versa. I'm not immediately sure how to resolve the problem though, or in any case a note about it seems inelegant and easy to miss.
```
$ docker system info -f '{{ .LoggingDriver }}'
json-file
```

**Reply by Alkarex** [reply]
We could maybe specify json-file explicitely, or find another more generic syntax that prevents logs from growing indefinitelly

**Reply by Frenzie** [reply]
According to https://docs.docker.com/config/containers/logging/local/ it defaults to 20m, suggesting we can just leave it off? I distinctly recall having seen Docker log files of hundreds of megabytes though.
